volume/lcow_parser.go

```diff
@@ -2,15 +2,14 @@ package volume
 
 import (
 	"errors"
-	"fmt"
 	"path"
 
 	"github.com/docker/docker/api/types/mount"
 )
 
 var lcowSpecificValidators mountValidator = func(m *mount.Mount) error {
 	if path.Clean(m.Target) == "/" {
```

Comment:
Wondering if this should also do the normalization step (`strings.Replace(p, `\`, `/`, -1)`), but not something for this PR